operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -121,6 +121,19 @@ local utils = import 'mixin-utils/utils.libsonnet';
       },
     },
 
+  // Returns the URL of a given dashboard, keeping the current time range and variables.
+  dashboardURL(filename)::
+    // Grafana uses a <base> HTML set to the path defined in GF_SERVER_ROOT_URL.
+    // This means that if we create relative links (starting with ".") the browser
+    // will append the base to it, effectively honoring the GF_SERVER_ROOT_URL.
+    //
+    // IMPORTANT: due to an issue with Grafana, this URL works only when opened in a
+    // new browser tab (e.g. link with target="_blank").
```

Comment:
Note to reviewers: I'm still talking to Grafana team about it. I will either reference an issue or change it, if they will come up with a working alternative.